src/openloco/graphics/gfx.cpp

```diff
@@ -98,6 +98,8 @@ namespace openloco::gfx
         // This code copies the closest variants into their place, and moves other elements accordingly
         if (header.num_entries == g1_expected_count::steam)
         {
+            elements.reserve(g1_expected_count::disc);
```

Comment:
`resize()`. `reserve()` will only change capacity so you can pre-emptively allocate the space, but vector's size stays the same and the objects you copy in will not survive any calls that modify (e.g. copying to another vector) or will be replaced with values pushed in future.

Reply:
Yes, sorry I didn't realise it needed to grow.